javascript-loops-practice/get-indexes.js

```diff
@@ -0,0 +1,10 @@
+/* exported getIndexes */
+function getIndexes(array) {
+  var indexes = [];
+  var i = array.length;
+  while (i > 0) {
+    indexes.unshift(i - 1);
+    i--;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.